package.json

```diff
@@ -21,6 +21,12 @@
       "pre-commit": "lint-staged"
     }
   },
+  "nodemonConfig": {
+    "verbose": true,
+    "watch": [
+      ".yalc/react-storefront/service-worker/[b]ootstrap.js"
```

Comment:
@kevhender What's with the "[b]" here?

Reply:
There's a weird bug in nodemon (https://github.com/remy/nodemon/issues/951) where watching a single file will only trigger the watch a single time, and then not work anymore after that. Apparently adding the `[b]` allows nodemon to continue watching the file after the first change.